src/rtctools_heat_network/workflows/grow_workflow.py

```diff
@@ -518,7 +521,7 @@ def solver_options(self):
 def main(runinfo_path, log_level):
     logger.info("Run Scenario Sizing")
     _ = run_optimization_problem(
-        EndScenarioSizingHIGHS,
+        EndScenarioSizing,
```

Comment:
let's put the default here back to highs

Reply:
Yes, good idea.